<!-- Driver launch file -->
<launch>
  <node pkg="bill_drivers" type="encoder_driver"
    name="encoder_driver">
  </node>

  <node pkg="bill_drivers" type="fan_driver"
    name="fan_driver">
  </node>

  <node pkg="bill_drivers" type="serial_driver"
    name="serial_driver" output="screen">
  </node>

  <node pkg="bill_drivers" type="led_driver"
    name="led_driver">
  </node>

  <node pkg="bill_drivers" type="motor_driver"
    name="motor_driver">
  </node>

  <node pkg="bill_drivers" type="reset_driver"
    name="reset_driver">
  </node>

  <node pkg="bill_drivers" type="ultrasonic_driver"
    name="ultrasonic_driver">
  </node>
</launch>
